lib/ansible/modules/windows/win_get_url.py

```diff
@@ -73,6 +73,20 @@
     type: bool
     default: yes
     version_added: '2.4'
+  checksum:
+    description:
+      - 'If a checksum is passed to this parameter, the digest of the
+        destination file will be calculated after it is downloaded to ensure
+        its integrity and verify that the transfer completed successfully.
+        Format: <algorithm>:<checksum|url>, e.g. checksum="sha256:D98291AC[...]B6DC7B97",
+        checksum="sha256:http://example.com/path/sha256sum.txt"'
+      - Additionally, if a checksum is passed to this parameter, and the file exist under
+        the C(dest) location, the I(destination_checksum) would be calculated, and if
+        checksum equals I(destination_checksum), the file download would be skipped
+        (unless C(force) is true).
+    type: str
+    default: ''
```

Comment:
Default was $null, which means unset. No need to add this.
```suggestion
```

Reply:
Fixed